lib/video_info/providers/youtube.rb

```diff
@@ -60,15 +65,15 @@ def _url_regex
       end
 
       def _api_base
-        "gdata.youtube.com"
+        "www.googleapis.com"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.